modules/mediawiki/manifests/extensionsetup.pp

```diff
@@ -1,6 +1,5 @@
-# MediaWiki extension setup
+# === Class mediawiki::extensionsetup
 class mediawiki::extensionsetup {
-
     ensure_packages('composer')
```

Comment:
Should be required by everything below

Reply:
This will ensure it is present before the below is ran. It shouldn't need explicitly required. I can if you want though?

Reply:
Should be ok